src/main/kotlin/ink/abb/pogo/scraper/Main.kt

```diff
@@ -30,16 +30,11 @@ fun getAuth(settings: Settings, http: OkHttpClient): CredentialProvider {
 
     val auth = if (username.contains('@')) {
         if (token.isBlank()) {
-            GoogleCredentialProvider(http, object : GoogleCredentialProvider.OnGoogleLoginOAuthCompleteListener {
-                override fun onInitialOAuthComplete(googleAuthJson: GoogleAuthJson?) {
-                }
-
-                override fun onTokenIdReceived(googleAuthTokenJson: GoogleAuthTokenJson) {
-                    Log.normal("Setting Google refresh token in your config")
-                    settings.setToken(googleAuthTokenJson.refreshToken)
-                    settings.writeProperty("config.properties", "token")
-                }
-            }, time)
+            GoogleAutoCredentialProvider(http, username, password)
```

Comment:
Use `GoogleAutoCredentialProvider(http, username, password,time)` here, the function without time is deprecated.